pkg/collector/corechecks/oracle/activity.go

```diff
@@ -181,6 +178,7 @@ func (c *Check) SampleSession() error {
 	} else {
 		activityQuery = activityQueryDirect
 	}
+	activityQuery = strings.ReplaceAll(activityQuery, "{sql_substr_length}", fmt.Sprintf("%d", maxSQLTextLength))
```

Comment:
We'll introduce a bind value (a parameter) to avoid having multiple versions of the statement (for each substring length) in the cache.